### PR TITLE
Announcement post for v0.75.0 RC1

### PR DESCRIPTION
**Pull request by dreamer**
Announcement post - @kcgen please take a look and tell me if I should add/modify something.

### CODE REVIEW

File: content/downloads/linux/index.md [hunk 1]
```diff
@@ -42,6 +42,21 @@ They depend on the following packages:
     sudo pacman -S sdl2 sdl2_net opusfile
 
 
+## <a name="steam"></a> Steam
+
+You can easily configure your DOS games on Steam to use dosbox-staging via
+[Boxtron](https://github.com/dreamer/boxtron) (community-developed
+Steam Play compatibility tool for DOS games).
```
Comment (kcgen) [inline content/downloads/linux/index.md:49]: If your Steam library contains DOS games, we recommend using [Boxtron](https://github.com/dreamer/boxtron) to easily configure them to run using dosbox-staging, among other settings.  Boxtron is is a  community-developed Steam Play compatibility-tool for DOS games.

File: content/downloads/linux/index.md [hunk 2]
```diff
@@ -42,6 +42,21 @@ They depend on the following packages:
...
+
+If you use dosbox-staging `dosbox` command is in your PATH,
+then Boxtron will pick it up automatically.  You can also
+configure Boxtron to use specific build by editing file
```
Comment (kcgen) [inline content/downloads/linux/index.md:53]: Boxtron will automatically use `dosbox` if found in your PATH or can be configured to use a specific binary by editing the file (...)

File: content/v0.75.0-rc1.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+This release is significant for several reasons:
+
+1. Our first stable build - from this point forward, builds named
+   0.75.x will only contain bugfixes.
+
```
Comment (kcgen) [inline content/v0.75.0-rc1.md:18]: It's our first stable release. From this point forward, the 0.75.x series will only contain bugfixes.

File: content/v0.75.0-rc1.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+2. Start of the new development cycle for the next release - we have a number
+   of planned features lined up already.
```
Comment (kcgen) [inline content/v0.75.0-rc1.md:20]: It kicks off the 0.76 development cycle, for which a number of features are already lined up.

File: content/v0.75.0-rc1.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+3. The first packages we offer for download through the new project home
+   page - users won't need to have GitHub account to access our builds 
+   any more.
```
Comment (kcgen) [inline content/v0.75.0-rc1.md:24]: The new project homepage makes the downloads publicly accessible, without the need for a GitHub account.

File: content/v0.75.0-rc1.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+to comb through. Hopefully, within a couple of days, we'll be able to write
+a nice summary with a description of new features, improvements, and bugfixes.
+
+In the meantime, we invite everyone to start testing RC1 builds; there are
+some rough edges for sure - if you find any bugs, please report them in
+[our bugtracker][bugtracker]. You can also ask questions on the
+**#dosbox&#8209;staging** channel in [Luxtorpeda Discord Server][discord].
+
+This release includes all upstream changes until SVN r4336, and 857 Git commits
+on top of that.
```
Comment (kcgen) [inline content/v0.75.0-rc1.md:36]: Just trying to sort out the numbers: ~1500 total commits; we have upstream and then 857 of our own - does that mean upstream did ~643 commits in the same timeframe?

**Review by kcgen — APPROVED** [review]
Nice announcement! Just a couple minor changes as usual.

File: content/v0.75.0-rc1.md (hunk 6 above)
Comment (dreamer) [reply content/v0.75.0-rc1.md:36]: Worse, upstream made that many commits to SVN over *last 10 years*:

```
$ git log --oneline svn/RELEASE_0_74..svn/trunk  | wc -l
689
```
That tag is 0.74 release; many of those 689 commits were backported to 0.74-2 and 0.74-3, but much fewer than I expected.

**Comment by dreamer** [comment]
Thanks for the review :) I'll reword the article before merging-in changes later today.

**Pull request merged**